### PR TITLE
Added a License Header to README.md.

### PR DESCRIPTION
**Pull request by rahulsunil2**
+ License Header which is common in all Microsoft Project Repositories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +33,9 @@ trademarks or logos is subject to and must follow
 [Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general).
 Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
 Any use of third-party trademarks or logos are subject to those third-party's policies.
+
+## License
+
+Copyright (c) Microsoft Corporation. All rights reserved.
+
+Licensed under the [MIT](LICENSE.md) license.
```
Comment (layershifter) [inline README.md:41]:
```suggestion
This project is licensed under the MIT License, see the [MIT](LICENSE.md) file for details.
```

Let's link the license instead

**Review by rahulsunil2 — COMMENTED** [review]
Made changes according to the suggestions and added it to the pull request.

File: README.md [hunk 2]
```diff
@@ -36,6 +36,4 @@ Any use of third-party trademarks or logos are subject to those third-party's po
 
 ## License
 
-Copyright (c) Microsoft Corporation. All rights reserved.
-
-Licensed under the [MIT](LICENSE.md) license.
+This project is licensed under the MIT License, see the [MIT](LICENSE.md) file for details.
```
Comment (layershifter) [inline README.md:39]:
```suggestion
This project is licensed under the MIT License, see the [LICENSE.md](LICENSE.md) file for details.
```

**Review by layershifter — APPROVED** [review]

**Comment by layershifter** [comment]
Thank you ❤️

**Comment by rahulsunil2** [comment]
Thank you so much for merging the changes. This is my first pull request on a Microsoft Repo. I'm so glad and excited that you approved my changes. I would love to contribute more. I'll try to work on something more towards programming than docs next time. 😊

**Pull request merged**